src/phongo_compat.h

```diff
@@ -22,6 +22,10 @@
 #include <Zend/zend_string.h>
 #include <Zend/zend_portability.h>
 
+/* Include stdbool.h as it might not have been implicitly loaded yet */
+#include <stdbool.h>
```

Comment:
Is this still relevant? I don't see `bool` used anywhere in this file. Just `zend_bool`.

Reply:
You are right - this was left over from a previous version. I've removed it.

Reply:
Correction: this is still necessary, at least on some versions of PHP. With PHP 8 or newer, `zend_bool` was changed to use a standard `bool` instead of a `unsigned char` (see https://github.com/php/php-src/commit/4b77a158ef2850582aeb4834c588aba49942776c). This adds an include for stdbool.h in zend_types.h, which we include in phongo_compat.h.

We can either leave the include in place here (to ensure we have stdbool.h available when needed on PHP < 8) or include it explicitly in every file that makes use of a `bool` type. I'm fine leaving it in phongo_compat.h for the time being.

Reply:
If https://github.com/php/php-src/commit/4b77a158ef2850582aeb4834c588aba49942776c is the first place `zend_bool` is defined, I don't understand how `stdbool.h` would not already be included by that point. We're already including `Zend/zend_types.h` at the top of this file.

I remember encountering this before but I never fully wrapped my head around it.

OK to leave as-is but we can consider opening a ticket to get to the bottom of it later.

Reply:
For PHP versions before 8.0 `zend_types.h` does not include `stdbool.h`, triggering the error there. Given that, I think including it in phongo_compat.h makes sense.